docs/source/requirements-single.rst

```diff
@@ -29,130 +29,86 @@ Managed node
 ============
 
 The managed z/OS node is the host that is managed by Ansible, as identified in
-the Ansible inventory.
-The managed node has dependencies that are specific to each release of the
-**IBM z/OS core collection**. Review the details of the dependencies before you
-proceed to install the IBM z/OS core collection.
+the Ansible inventory. For the **IBM z/OS core collection** to manage the z/OS node,
+some dependencies are required to be installed on z/OS such as:
 
-* z/OS `V2R3`_ or `later`_
+* `z/OS`_
```

Comment:
I see the 'V2R3' of z/OS was removed, and I don't see it anywhere else on the page.  Is this intentional?  That's my only concern.

Reply:
yea, we're pointing users to the release notes for specific release versions rather than keeping them here
eg for 1.8.0-beta.1:
![image](https://github.com/ansible-collections/ibm_zos_core/assets/10567394/ad5186ad-326b-47e1-8a06-8232312f4fcb)

https://ibm.github.io/z_ansible_collections_doc/ibm_zos_core/docs/source/release_notes.html#reference